SpywareFilter/sections/specific.txt

```diff
@@ -81,8 +82,8 @@
 ||stats.bio.link/api/event/pageView?
 ||shopify.com/__dux|
 ||shopify.com/__dux|$ping,redirect-rule=nooptext
-/shopify/monorail/*/produce$domain=runpacers.com|aquariumcoop.com|online.citysuper.com.hk|windsorstore.com|clothesmentor.com|fashionnova.com|lttstore.com
-/shopify/monorail/*/produce$ping,redirect-rule=nooptext,domain=runpacers.com|aquariumcoop.com|online.citysuper.com.hk|windsorstore.com|clothesmentor.com|fashionnova.com|lttstore.com
+/shopify/monorail/*/produce$domain=runpacers.com|aquariumcoop.com|online.citysuper.com.hk|windsorstore.com|clothesmentor.com|fashionnova.com|lttstore.com|mrbeast.store
```

Comment:
I think that we can generalize the rules to the following rule:
```adblock
/.well-known/shopify/monorail
```

It also exists in EasyPrivacy.